### PR TITLE
update kubernetes page with logos and video

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* added a logo cloud with a funky Local host with icon and text (needs design review)
* added a vimeo video

## QA

1. go to http://www.ubuntu.com-update-kubernetes-with-logos.demo.haus/cloud/kubernetes
2. see the new logo cloud section
3. see the new video
4. make sure it looks good in S/M/L

## Issue / Card

[trello card](https://trello.com/c/HezAAfDf)
[copy doc](https://docs.google.com/document/d/1VMu4i8rJHW7BXKiOiy36XVFsihVxrNEMzUHn3X2XabI/edit#)

### CODE REVIEW

**Comment by pmahnke** [comment]
@joanasa89 can you review as well?

**Comment by pmahnke** [comment]
[Update Kubernetes page](https://trello.com/c/HezAAfDf/2245-update-kubernetes-page)

File: templates/cloud/kubernetes.html [hunk 1]
```diff
@@ -111,5 +165,5 @@
 {% endblock content %}
 
 {% block footer_extra %}
-    <script async src="https://assets.ubuntu.com/v1/juju-cards-v1.3.0.js"></script>
+    <script async src="{{ ASSET_SERVER_URL }}juju-cards-v1.3.0.js"></script>
```
Comment (nottrobin) [inline templates/cloud/kubernetes.html]: :heart:

**Comment by joanasa89** [comment]
Change local hosts logo to : https://assets.ubuntu.com/v1/04583476-local+host.svg

**Pull request merged**